src/controllers/playback/videoosd.js

```diff
@@ -1280,7 +1294,7 @@ define(["playbackManager", "dom", "inputManager", "datetime", "itemHelper", "med
                 showOsd();
                 inputManager.on(window, onInputCommand);
                 dom.addEventListener(window, "keydown", onWindowKeyDown, {
-                    passive: true
+                    capture: true
```

Comment:
why?

Reply:
to `stopPropagation` after OSD hide before `keyboardnavigation` make "go to back"

Reply:
Oh, right, I thought `passive` did the opposite thing, like "this element should be passive, JS function does all the stuff" LOL